script/eval.py

```diff
@@ -0,0 +1,21 @@
+from fire import Fire
+from metrics import mse_per_sample, psnr_per_sample, ssim_per_sample
+
+def evaluation_script(y_test, trained_model):
+    y_pred = trained_model(eval_data)
+    mse_value = mse_per_sample(y_pred, y_test)
+    psnr_value = psnr_per_sample(mse_value)
+    ssim_value = ssim_per_sample(y_pred, y_test)
```

Comment:
we can actually use validate.py which will calculate these metrics for us. it would also save produced images as well.